test/cli/daemon.js

```diff
@@ -9,10 +9,35 @@ const os = require('os')
 const path = require('path')
 const hat = require('hat')
 const fs = require('fs')
+const tempWrite = require('temp-write')
 const pkg = require('../../package.json')
 
 const skipOnWindows = isWindows() ? it.skip : it
+const daemonReady = (daemon, fn) => {
```

Comment:
what is this fn? A callback? mind calling it a callback?

Reply:
its a function to be called after ready not a callback

Reply:
and checking if it is ready, is an async operation because it has to do IO, hence it is a callback to return to the original routine once it is ready